ci/roles/build_containers/defaults/main.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-# Path of the authentication file. Use REGISTRY_AUTH_FILE environment variable to override. (default: /root/containers/auth.json)
-# authfile_path
+# Path of the authentication file. Use REGISTRY_AUTH_FILE environment variable to override.
+authfile_path: ${XDG_RUNTIME_DIR}/containers/auth.json
```

Comment:
what/where is setting this XDG_RUNTIME_DIR ?

Reply:
${XDG_RUNTIME_DIR} is a env variable. The path automatically gets created on doing podman/buildah login.
More info is here: https://docs.podman.io/en/latest/markdown/podman-login.1.html#description
```
❯ echo ${XDG_RUNTIME_DIR}
/run/user/1000
```

Reply:
thank you